### ISSUE TITLE
update to latest version (supporting linux natively)

### ISSUE THREAD

**Issue by CircleCode** [sanderboom/aur-kitematic#1]
as stated by  oblique on 2015-12-07 07:04 at https://aur.archlinux.org/packages/kitematic/,

> Linux support was merged in official git repo a week ago. Please update.
> 
> Thanks!

Do you plan to upgrade to this version ?

**Comment by sanderboom** [comment]
Yes definitely - I'll see if I can find some time.

**Comment by CircleCode** [comment]
wonderful, thanks

**Comment by sanderboom** [comment]
Did you find any .deb / .rpm packages for Kitematic?

**Comment by CircleCode** [comment]
not for now, there's only https://github.com/docker/kitematic/pull/696#issuecomment-161389916 saying that so is working on a ppa

**Comment by CircleCode** [comment]
and it seems there won't be any official package soon like said in https://forums.docker.com/t/why-do-you-not-support-development-on-linux/4833/2

**Comment by sanderboom** [comment]
Upgraded version in AUR, let me know if it works!

**Comment by CircleCode** [comment]
for me, kitematic start, but complains not being able to talk with the daemon, thus supposing it is not running. I have no idea why…

> Cannot connect to the Docker daemon. The daemon is not running.

**Comment by sanderboom** [comment]
Hmm
- Is your docker daemon running?
- Under what user is the daemon running?
- Do you start Kitematic as the same user?

**Comment by CircleCode** [comment]
After investigations, the problem came from my systemd unit file (I had an obsolete snippet replacement)
It works fine now, thanks

**Comment by CircleCode** [comment]
the last problem seems to be when the application is closed: the ihm closes, but the process stays alive

**Comment by sanderboom** [comment]
Yes, I have the same. I addressed this earlier, see: https://github.com/docker/kitematic/issues/49#issuecomment-141643232

The answer: https://github.com/docker/kitematic/issues/49#issuecomment-147677883
